libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/KaptIncrementalWithAggregatingApt.kt

```diff
@@ -275,8 +275,9 @@ class KaptIncrementalWithAggregatingApt : KaptIncrementalIT() {
             assertEquals(
                 setOf(
                     fileInWorkingDir("build/tmp/kapt3/stubs/main/bar/NoAnnotationsKt.java").canonicalPath,
-                    fileInWorkingDir("build/tmp/kapt3/stubs/main/error/NonExistentClass.java").canonicalPath
-                ), getProcessedSources(output)
+                    fileInWorkingDir("build/tmp/kapt3/stubs/main/error/NonExistentClass.java").canonicalPath,
+                    fileInWorkingDir("build/generated/source/kapt/main/bar/WithAnnotationGenerated.java").canonicalPath.takeUnless { isBinary },
+                ).filterNotNull().toSet(), getProcessedSources(output)
```

Comment:
You can use `setOfNotNull()`

Reply:
Tests are compiled with apiVersion 1.3 and `setOfNotNull` got added in 1.4.